### PR TITLE
WCMS-19787 added modified date information to /data page

### PR DESCRIPTION
**Pull request by averysmithproductions**
**Description**

Dataset pages and the filtered view page (/data) display the modified date in the top right corner below the dataset title.

**AC**

Display the modified date in the top right corner of the dataset page as "Updated March 27, 2024"
Also display on the /data page.

**Before**

Screenshot 2024-04-01 at 5 05 44 PM

Screenshot 2024-04-01 at 5 06 27 PM

**After**

Screenshot 2024-04-02 at 11 35 52 AM

Screenshot 2024-04-01 at 5 06 47 PM

**To Test**

1. View a dataset page, ie: 
- http://localhost:3000/dataset/ebcfc16f-8291-4c61-82a4-055846d72f3a
- http://localhost:3000/dataset/ebcfc16f-8291-4c61-82a4-055846d72f3a/data
2. Confirm you see the modified date in the top right of the content region.

### CODE REVIEW

File: src/types/dataset.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export type DatasetType = {
   identifier: string,
   describedBy?: any, // TODO
   describedByType?: any, //TODO
+  modified: string,
```
Comment (brdunfield) [inline src/types/dataset.ts:23]: This has added some typescript errors to /src/services/useMetastoreDataset.tsx

**Review by brdunfield — CHANGES_REQUESTED** [review]
The ticket requests 8pt gap between title and date, and another 8pt gap between date and description.
The /data page looks fine, but the regular dataset needs to be revised to meet this request.

**Comment by averysmithproductions** [comment]
@brdunfield 

> The ticket requests 8pt gap between title and date, and another 8pt gap between date and description. The /data page looks fine, but the regular dataset needs to be revised to meet this request.

addressed

https://github.com/GetDKAN/cmsds-open-data-components/pull/182/commits/e5c7583e868ae703a23dac5237389e3cab486735

File: src/types/dataset.ts (hunk 1 above)
Comment (averysmithproductions) [reply src/types/dataset.ts:23]: feedback addressed:

https://github.com/GetDKAN/cmsds-open-data-components/pull/182/commits/b70571dfa474f49746574035eaf652ac6d696064

**Review by brdunfield — APPROVED** [review]
🦑

**Pull request merged**